website/content/en/preview/upgrade-guide/_index.md

```diff
@@ -97,6 +97,9 @@ By adopting this practice we allow our users who are early adopters to test out
 
 # Released Upgrade Notes
 
+## Upgrading to v0.18.0+
+* v0.18.0 removes the `karpenter_consolidation_nodes_created` and `karpenter_consolidation_nodes_terminated` prometheus metrics in favor of the more generic `karpenter_nodes_created` and `karpenter_nodes_terminated` metrics. You can still see nodes created and terminated by consolidation by checking the `reason` label on the metrics. Check out all the metrics published by Karpenter [here](../tasks/metrics/).
```

Comment:
Reading this it makes me wish we'd chosen more polar terms, e.g. created/deleted, launched/terminated. It's nonzero work and not massively valuable, so happy to move forward as is.